City-of-Helsinki/drupal-helfi-tyo-yrittaminen
# UHF-3321 Update elo instance

How to test:

1. Checkout this branch
2. `make drush-cr && make drush-cim`
3. Go to create a basic page with liftup with image paragraph and add a button to it. Or you can add button to any wysiwyg content and save the page. Make sure that the button looks good and not too small.
4. Also make sure that there is no TPR stuff on.
